python/paddle/distributed/auto_parallel/api.py

```diff
@@ -142,3 +141,32 @@ def shard_tensor(
         return shard_tensor_static(
             data, dist_attr.process_mesh, dist_attr.sharding_specs
         )
+
+
+def dtensor_from_fn(fn, dist_attr, *args, **kwargs):
+    """
+    Construct a Distributed Tensor from a function of arguments.
+
+    Args:
+        fn (callable): A callable function that takes arguments of Distributed Tensor and returns tensor.
+        dist_attr (paddle.distributed.DistAttr): Specify how tensors are distributed or sliced on ProcessMesh.
+        *args (tuple): A tuple of arguments to be passed to the ``fn`` function.
+        **kwargs (dict): A dict of arguments to be passed to the ``fn`` function.
+
+    Retruns:
+        Tensor: A Tensor constructed from ``fn`` with distributed attributes.
+
+    Examples:
+
+        .. code-block:: python
+            >>> import paddle
```

Comment:
```suggestion
        .. code-block:: python
        
            >>> import paddle
```
code-block 下加个空行，否则无法正常预览代码块

Reply:
done， thx